setup.json

```diff
@@ -83,8 +83,11 @@
       "markupsafe==1.1.1",
       "pygments==2.4.2",
       "sphinx-rtd-theme==0.4.3",
-      "sphinx==1.8.5",
-      "sphinxcontrib-contentui==0.2.2"
+      "sphinx==1.8.5; python_version<'3'",
+      "sphinx==2.2.0; python_version>='3.0'",
+      "sphinxcontrib-contentui==0.2.2; python_version<'3'",
+      "sphinxcontrib-contentui==0.2.4; python_version>='3.0'",
```

Comment:
@ltalirz this is the main change compared to before: For the Python3 version, Sphinx needed to be updated (to at least 2.0) to be compatible with ``sphinxcontrib-details-directive``, and that in turn required an update to ``sphinxcontrib-contentui``.